### PR TITLE
10198 add module website_captcha_nogoogle

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by elicoidal** [comment]
Propose directly to OCA
Eric CAUDAL --
_Eric Caudal_ /[Founder and CEO]/
Skype: elico.corp. Phone: + 86 186 2136 1670 (Cell), + 86 21 6211 
8017/27/37 (Office)
_Elico Shanghai_ (Hong Kong/Shenzhen/Singapore) 
http://www.elico-corp.com /_Odoo Gold Partner, best Odoo Partner 2014 
for APAC_/

**Comment by Rona111** [comment]
cc @elicoidal  @duanyp1991 @seb-elico @siyuancn

File: website_captcha_nogoogle/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    'name': 'Website Captcha',
+    'version': '8.0.1.0.0',
+    'category': 'Website',
+    'depends': [
+        'website',
+    ],
+    'author': 'Elico Corp',
+    # 'license': 'LGPL-3',
+    'website': 'https://www.elico-corp.com',
+    'currency': 'EUR',
```
Comment (elicoidal) [inline website_captcha_nogoogle/__openerp__.py]: Remove price and currency

**Comment by siyuancn** [comment]
What's the usage for image: icon.png?

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: website_captcha_nogoogle/README.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Website Captcha No Google
```
Comment (elicoidal) [inline website_captcha_nogoogle/README.rst:6]: Website Captcha (No Google)

File: website_captcha_nogoogle/__init__.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import website
```
Comment (elicoidal) [inline website_captcha_nogoogle/__init__.py:5]: Please sort in alphabetical order following: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

File: website_captcha_nogoogle/doc/index.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
+Odoo CAPTCHA (No Google)
```
Comment (elicoidal) [inline website_captcha_nogoogle/doc/index.rst:1]: Should be directly included in the README file

**Comment by noahzaozao** [comment]
LGTM

**Pull request merged**